pep2html.py

```diff
@@ -579,11 +579,11 @@ def get_pep_type(input_lines):
             # End of the RFC 2822 header (first blank line).
             break
         elif line.startswith('content-type: '):
-            pep_type = line.split()[1] or 'text/plain'
+            pep_type = line.split()[1] or 'text/x-rst'
             break
         elif line.startswith('pep: '):
             # Default PEP type, used if no explicit content-type specified:
-            pep_type = 'text/plain'
+            pep_type = 'text/x-rst'
```

Comment:
GH won't let me do a suggestion, but delete these three lines if you go with my others in this file.

Reply:
See above